### ISSUE TITLE
Implement all ansible playbooks as task on website

### ISSUE THREAD

**Issue by RoboPython** [RoboPython/flaskStarter#6]

**Comment by RoboPython** [comment]
I don't think we need to worry about this too much. As toby specifically said just get a subset of it working.
So if get version and pull local copy work perfectly then fine and maybe we could add one or two more

**Comment by Hoolean** [comment]
Owing to the completion and generalisation of Playbooks to the code - with the creation and usage of playbooks.json - this can now go ahead. :aerial_tramway:
